newIDE/app/src/VariablesList/VariablesList.js

```diff
@@ -219,16 +229,32 @@ const VariablesList = ({ onComputeAllVariableNames, ...props }: Props) => {
 
   React.useEffect(
```

Comment:
for our future self, could you add a small comment above each useEffect to easily understand what is responsible for what?